client/cypress/e2e/proposals.cy.ts

```diff
@@ -139,6 +140,38 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight }) =>
       removeProposalFromAllocate(proposalNames.length, 2, 0);
       removeProposalFromAllocate(proposalNames.length, 1, proposalNames.length - 1);
     });
+
+    it('ProposalsTimelineWidgetItem with href opens link when clicked without mouse movement', () => {
+      const milestones = getMilestones();
+      cy.get('[data-test=ProposalsTimelineWidget]').should('be.visible');
+      cy.get('[data-test=ProposalsTimelineWidgetItem]').should('have.length', milestones.length);
+      for (let i = 0; i < milestones.length; i++) {
+        if (milestones[i].href) {
+          cy.get('[data-test=ProposalsTimelineWidgetItem]')
+            .eq(i)
+            .within(() => {
+              cy.get('[data-test=ProposalsTimelineWidgetItem__Svg--arrowTopRight]').should('exist');
```

Comment:
Shouldn't it be `be.visible`? Exist doesn't mean the element is actually visible to the user.
```
cy.get('[data-test=ProposalsTimelineWidgetItem__Svg--arrowTopRight]').should('be.visible');
```

Reply:
Changed